check_pgactivity

```diff
@@ -5489,7 +5489,24 @@ sub check_longest_query {
                 WHERE state = 'active'
             ) AS s ON (d.datname=s.datname)
             WHERE d.datallowconn
-        }
+        },
+	$PG_VERSION_100 => q{SELECT d.datname,
+                COALESCE(elapsed, 0),
+                COALESCE(query, ''),
+                application_name
+            FROM pg_database AS d
+            LEFT JOIN (
+                SELECT datname, query,
+                    extract('epoch' FROM
+                        date_trunc('second', current_timestamp-state_change)
+                    ) AS elapsed,
+                    application_name
+                FROM pg_stat_activity
+                WHERE state = 'active'
+		  AND backend_type IN ('client backend', 'background worker', 'autovacuum worker')
```

Comment:
I wonder if we should remove autovacuum worker here.

There's already a service relating to autovacuum, this check is about is about long running user query and autovacuum is designed to be slowly-ish, in the background anyway.

Reply:
I asked myself this question, but didn't think about the redundancy with the autovacuum specific check. Let's remove them and add a comment suggesting to use the autovacuum service.